.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: markdownlint-cli2
+    name: markdownlint-cli2
+    description: "Checks the style of Markdown/Commonmark files."
+    entry: markdownlint-cli2
+    language: node
+    types: [markdown]
+    minimum_pre_commit_version: 0.15.0
```

Comment:
Why is this needed?

Reply:
`types` property [has been added in version `0.15.0`](https://github.com/pre-commit/pre-commit/blob/master/CHANGELOG.md#0150---2017-07-02).